CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-- [...]
+- **[BREAKING CHANGE]** - Replaced `fullscreen` prop from `Modal` with new entry in enum `ModalSize`
```

Comment:
by a new entry?

Reply:
nope `with` is the correct way to say it in english :)